python/plugins/db_manager/db_plugins/gpkg/connector.py

```diff
@@ -59,8 +59,10 @@ def _opendb(self):
         self.gdal_ds = gdal.OpenEx(self.dbname, gdal.OF_UPDATE)
         if self.gdal_ds is None:
             self.gdal_ds = gdal.OpenEx(self.dbname)
-        if self.gdal_ds is None or self.gdal_ds.GetDriver().ShortName != 'GPKG':
+        if self.gdal_ds is None:
             raise ConnectionError(QApplication.translate("DBManagerPlugin", '"{0}" not found').format(self.dbname))
+        if self.gdal_ds.GetDriver().ShortName != 'GPKG':
+            raise ConnectionError(QApplication.translate("DBManagerPlugin", '"{dbname}" not recognized as GPKG ({shortname} reported instead.)').format(dbname=self.dbname, shortname=self.gdal_ds.GetDriver().ShortName))
```

Comment:
Question: why not use 
```suggestion
            raise ConnectionError(QApplication.translate("DBManagerPlugin", '"%1" not recognized as GPKG (%2 reported instead.)').format(self.dbname, self.gdal_ds.GetDriver().ShortName))
```
or something like that? i'm afraid that {shortname} gets translated (found these errors in the past)

Reply:
I'm fine with what was done using the `{}` syntax. It's something that I wanted to see more in QGIS code. This is much better for translators to have some context.

Should we warn translators about guidelines? 
https://github.com/inasafe/inasafe/wiki/Translation-Guidelines rule number 1.

Reply:
BTW @DelazJ it's `{}`, not `%1` or `%2` in your suggested change.

Reply:
It would be possible to use `{0}` and ´{1}´, I find the other more readable but translators need to be aware.
The inasafe guidelines look really nice! Would be great to put them into QGIS too.

Reply:
OK. Let's move with the current approach ten.
>  The inasafe guidelines look really nice! Would be great to put them into QGIS too.

Yes but it would not be enough imo. Having it (at least) as a welcome message when joining a group in Transifex may be helpful.

Reply:
> Yes but it would not be enough imo. Having it (at least) as a welcome message when joining a group in Transifex may be helpful.

Does that work?

Reply:
Sorry @m-kuhn I forgot to reply. I'm not sure i understand your question as the solution is not implemented yet (maybe something to think/write/implement at A Coruna?).

Reply:
The question was if it's possible to show a welcome message on transifex. It would just need someone to look into it, in A Coruña or somewhere else.